### PR TITLE
Swedish translation

### PR DESCRIPTION
**Pull request by jony0008**
Please merge Swedish translation of stretchly.

### CODE REVIEW

File: app/locales/sv.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    "notificationsTitle": "",
+    "notificationsText": "Avisering visas 10 sekunder före mikrorast och 30 sekunder före rast, så du kan undvika frestelsen att hoppa över pausen eftersom du <i>verkligen-behöver-bara-30-sekunder-för-att-klara-detta</i>.",
+    "tray1Title": "",
+    "tray1Text": "Vill du se när nästa rast kommer? Håll muspekaren över ikonen i systemfältet.",
+    "tray2Title": "",
+    "tray2Text": "Är ditt flöde annorlunda? Ska du presentera något? Dröj stretchly för en tidsperiod. Behöver du en snabb rastidé? Klicka på systemfältsmenyn och du kan göra allt det. Även återställa raster som du just har börjat.",
+    "settings1Title": "",
+    "settings1Text": "Om du märker att du ständigt hoppa över raster, ändra inställningar och välj tid och raster som passar dig. Eller sätt på så kallat "striktläge" där du inte får hoppa över raster. Din kropp och ditt sinne kommer att tacka dig. ",
```
Comment (hovancik) [inline app/locales/sv.json]: can you use single quotes instead of double quotes? this would give errors

**Comment by hovancik** [comment]
Hi and many thanks ;] 

Did you have chance to run it locally?

File: app/locales/sv.json (hunk 1 above)
Comment (jony0008) [reply app/locales/sv.json]: Yes

File: app/locales/sv.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    "settings1Title": "",
+    "settings1Text": "Om du märker att du ständigt hoppa över raster, ändra inställningar och välj tid och raster som passar dig. Eller sätt på så kallat 'striktläge' där du inte får hoppa över raster. Din kropp och ditt sinne kommer att tacka dig. ",
+    "settings2Title": "",
+    "settings2Text": "Vi älskar alla färgscheman och ljud för slutet av pausen, så vi kunde inte bestämma vilken som skulle sättas som standard. Hjälp oss och välj din favorit, så de inte glöms.",
+    "settings3Title": "",
+    "settings3Text": "På sista inställningssidan kan du ändra språk för gränssnittet och några extra inställningar. Du behöver nog inte ändra dem, men hej, varför inte kolla in dem?",
+    "thanksTitle": "Tack!",
+    "thanksText": "Varje månad tar det mycket arbete att påminna dig om att ta en paus. Vi hoppas att stretchly hjälper dig att leda ett mer balanserat liv. Om det är så att du kan stödja mig med "Bli en beskyddare (Patron)"-knappen i systemfältsmenyn."
```
Comment (hovancik) [inline app/locales/sv.json]: One more here ;]  Also, please add yourself to readme: https://github.com/hovancik/stretchly#contributors 

You could also update CHANGELOG. If not sure, I can do it later.

**Comment by hovancik** [comment]
@jony0008 sorry but one more change that I forgot to mention: 

please edit settings3.html and welcome.html. You will need to add Swedish language there ;] That should be it!

**Pull request merged**